darwin/future/tests/meta/objects/test_stagemeta.py

```diff
@@ -81,3 +81,15 @@ def test_move_attached_files_to_stage(
             + f"v2/teams/default-team/items/ids?workflow_stage_ids={str(stage_meta.id)}&dataset_ids=1337",
             1,
         )
+
+def test_get_stage_id(stage_meta):
+    assert stage_meta.id == UUID("00000000-0000-0000-0000-000000000000")
+
+def test_get_stage_name(stage_meta):
+    assert stage_meta.name == "test-stage"
+
+def test_get_stage_type(stage_meta):
+    assert stage_meta.type == "annotate"
+
+def test_get_stage_edges(stage_meta):
```

Comment:
I'm concerned about the `test_get_stage_edges` test. The test uses the `stage_meta` fixture, which has `edges=[]`, but this doesn't actually check that the code in the edges property works, since it loops over each set of IDs within edges

Should the fixture be adjusted to include a more fleshed out structure of UUIDs, or should a new edges fixture be defined that can be passed into `stage_meta`, or is the test fine as is?

Reply:
Try making the test instantiate an object of type `Stage` and then setting and confirming the bits you've added, to show they carry.  It's not going to do much, but it confirms that you've done the change the ticket requires.

```python
def test_confirm_stage_edges():
    test_stage = Stage(
        # Other members set above
        edges=[
            [], # put edge data in this
            [], # and this
        ]
    )

    assert len(test_stage.edge) == 2
    # some other checks of the data here maybe
```

Reply:
Yeah, looking at that, that's really good as an improvement.  Nicely done.